packages/ui/src/common/Accordion/AccordionItem.tsx

```diff
@@ -0,0 +1,235 @@
+	transition: opacity 0.2s ease;
+	background: transparent;
+	border: none;
+	cursor: pointer;
+	padding: 0;
+	display: inline-flex;
+	align-items: center;
+
+	svg {
+		border-bottom: 2px solid ${theme.colors.secondary};
+	}
+
+	&:hover {
+		opacity: 1;
+	}
+`;
+
+const descriptionWrapperStyle = (theme: Theme) => css`
+	${theme.typography?.label2};
+	color: ${theme.colors.grey_5};
+	padding: 4px 8px;
+	word-wrap: break-word;
+	overflow-wrap: break-word;
+`;
+
+const accordionCollapseStyle = (isOpen: boolean) => css`
+	overflow: hidden;
+	max-height: ${isOpen ? 'none' : '0px'};
+	transition: max-height 0.3s ease;
+`;
+
+const accordionItemContentStyle = css`
+	padding: 30px;
+	background-color: #ffffff;
+`;
+
+const contentInnerContainerStyle = (theme: Theme) => css`
+	border-left: 1px solid ${theme.colors.grey_3};
+	padding-left: 30px;
+	${theme.typography?.data};
+`;
+
+const handleInitialHashCheck = (
+	windowLocationHash: string,
+	accordionData: AccordionData,
+	openState: AccordionOpenState,
+	indexString: string,
+) => {
+	if (window.location.hash === windowLocationHash) {
+		if (!accordionData.openOnInit) {
+			openState.toggle();
+		}
+		document.getElementById(indexString)?.scrollIntoView({ behavior: 'smooth' });
+	}
+};
+
+const hashOnClick = (
+	event: MouseEvent<HTMLButtonElement>,
+	windowLocationHash: string,
+	setClipboardContents: (currentSchema: string) => void,
+) => {
+	event.stopPropagation();
+	window.location.hash = windowLocationHash;
+	setClipboardContents(window.location.href);
+};
+
+const AccordionItem = ({ index, accordionData, openState, setClipboardContents }: AccordionItemProps) => {
+	const theme = useThemeContext();
+	const { description, title, content, dictionaryDownloadButtonProps } = accordionData;
+	const { ChevronDown, Hash } = theme.icons;
+
+	const indexString = index.toString();
+	const windowLocationHash = `#${index}`;
+
+	useEffect(() => {
+		handleInitialHashCheck(windowLocationHash, accordionData, openState, indexString);
+	}, []);
+
+	return (
+		<li role="button" css={accordionItemStyle(theme)} id={indexString} onClick={openState.toggle}>
+			<h2 css={accordionItemTitleStyle}>
+				<div css={contentContainerStyle}>
+					<button type="button" css={accordionItemButtonStyle(theme)}>
+						<ChevronDown
+							fill={theme.colors.accent_dark}
+							width={16}
+							height={16}
+							style={chevronStyle(openState.isOpen)}
+						/>
+						<span css={titleStyle}>{title}</span>
+					</button>
+					<button
+						type="button"
+						css={hashIconStyle(theme)}
+						onClick={(event) => hashOnClick(event, windowLocationHash, setClipboardContents)}
+					>
+						<Hash width={20} height={20} fill={theme.colors.secondary} />
+					</button>
+					{description && (
+						<ReadMoreText maxLines={MAX_LINES_BEFORE_EXPAND} wrapperStyle={descriptionWrapperStyle}>
+							{description}
+						</ReadMoreText>
+					)}
+				</div>
+				<DictionaryDownloadButton {...dictionaryDownloadButtonProps} />
```

Comment:
we can probably get away with just passing in a single schema name prop for this.
We'd have a specific instance of the DictionaryDownloadButton button with most of it's configs set because url, type and a bunch of other things won't change between these accordian types